### DISCUSSION TITLE
env var in post processor with regex

### DISCUSSION THREAD

**Discussion by davidwallis** [orhun/git-cliff#622] [General]
from what I can see the env vars don't work within replace command? (struggling to find examples)

currently I have:
```
{ pattern = "(([a-f0-9]{7})[a-f0-9]*)", replace = "[${2}](https://myserverl/xyz/abc/foo/_git/foo/commit/${1})" },
```
but I'd like to use the env var BUILD_REPOSITORY_URI

The ability to do this would make this config portable across my repo's without needing to template, but not sure if I'm missing something obvious for doing this.

Thanks

**Comment by orhun** [comment]
> from what I can see the env vars don't work within replace command? (struggling to find examples)

Yes, they are not directly supported now, but you can use them as in #610 

However, it might still not fit your use case. We are discussing how to improve this in #611 (ideas welcome!)

Also, this feature needs some documentation updates, see #609
